packages/design-tokens/src/scss/themes/light-inverted/_colors.scss

```diff
@@ -0,0 +1,275 @@
+// Generated Colors from Supernova. Do not edit manually.
+// Manually created from Figma
+$text-primary: #ffffff !default;
+$text-secondary: #e9e9e9 !default;
+$text-tertiary: #d6d6d6 !default;
+
+$emotion-danger-active: #ffffff !default;
+$emotion-danger-background: #ba3e5a !default;
+$emotion-danger-border: #ffffff !default;
+$emotion-danger-content: #e84b6f !default;
+$emotion-danger-hover: #ffffff !default;
```

Comment:
Emotion, button and custom colors mix interaction states with properties:

- states: active, hover
- properties: background, border, content

Looking at current tokens, it's unclear to me where I can use the `active` and `hover` tokens.
Don't we need an indication what the states can be used for? Something like:

```scss
$emotion-danger-background-active
$emotion-danger-background-hover
```